### ISSUE TITLE
ExGAN: Adversarial Generation of Extreme Samples

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#133]
# TL;DR
While GANs usually train and generate only typical data, ExGANs can generate low-probability data in a controlled manner; only the extremes of the samples generated by GANs are added to the dataset to bias the data, and then use them to train the GAN conditional on the extreme value. They confirmed that they can generate the data successfully in the weather data.
![image](https://user-images.githubusercontent.com/24524018/95159039-03063180-07d8-11eb-9c6d-87a94929216b.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.08454

# Submission Dates(yyyy/mm/dd)
2020/09/17

# Authors and institutions
Siddharth Bhatia, Arjit Jain, Bryan Hooi
- National University of Singapore
- IIT Bombay

# Methods

# Results

# Comments
